docutils-stubs/nodes.pyi

```diff
@@ -38,7 +39,7 @@ def ensure_str(s: str) -> str: ...
 
 class Text(Node, str):
     tagname: str = ...
-    children: Tuple[nodes.Node, ...] = ...
+    children: Tuple[T_co, ...] = ...
```

Comment:
By default, `Tuple` is covariant because it is immutable, while `List` is mutable.
So, this change is not necessary.

For more detail, refer following comment and PEP,

* https://github.com/python/typing/issues/2#issuecomment-69405519
* https://www.python.org/dev/peps/pep-0484/#covariance-and-contravariance
  "The read-only collection classes in typing are all declared covariant in their type variable (e.g. Mapping and Sequence)."